docs/v3/core/index.rst

```diff
@@ -880,7 +901,10 @@ should be interpreted.
 
 This core specification defines a limited set of data types to
 represent boolean values, integers, and floating point
-numbers. Extensions may define additional data types. All of the data
+numbers. These can be found under :ref:`Data Types<data-types-list>`.
+
+:ref:`Extensions<extensions_section>` may define additional
+data types. All of the data
```

Comment:
this applies to the old text, but the additional data type _is_ the extension. An additional data type is not defined by an extension -- it's defined by a spec.